lib/sensu/api/process.rb

```diff
@@ -382,7 +383,7 @@ def resolve_event(event_json)
                   settings.redis.smembers("history:#{client_name}") do |checks|
                     checks.each do |check_name|
                       settings.redis.del("history:#{client_name}:#{check_name}")
-                      settings.redis.del("execution:#{client_name}:#{check_name}")
+                      settings.redis.del("result:#{client_name}:#{check_name}")
```

Comment:
@solarkennedy this is the deletion @roobert mentioned, removing the latest check result.